standards/8x.md

```diff
@@ -0,0 +1,111 @@
+# Specification
+
+## Summary
+A standardized interface for interacting with recurring subscriptions.
+
+##  Abstract
+This standard defines a contract interface for interacting with subscription data. The 8x suggestions come under the feelings that implementation of how a subscription is created, is left up to the implementor. If we define a standard for implementation, we'll be locking ourselves into one way of doing things.
```

Comment:
i'm not sure i understand this.  this is phrased differently than the comment in the main PR.

you are proposing that the scope of the standard be the read-only components?  that will create an environment in which everyone who implmements a saas model has to learn different methods for each time they want to bill a customer.  seems llike an unnecessary bottleneck on the interoperability of the system